pkg/components/imguploader/webdavuploader.go

```diff
@@ -55,7 +56,7 @@ func (u *WebdavUploader) Upload(ctx context.Context, pa string) (string, error)
 	filename += pngExt
 	url.Path = path.Join(url.Path, filename)
 
-	imgData, err := ioutil.ReadFile(pa)
+	imgData, err := ioutil.ReadFile(filepath.Clean(pa))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue and any security concerns around the folder path should be resolved when originally constructing it.